common-libs/did-auth-lib/README.md

```diff
@@ -1,36 +1,193 @@
 # Affinidi Did Auth Helpers (PUBLIC)
 
 ## Usage:
+
+This library allows proving that the client/user is the owner of provided `DID`. Based on this proof App builders can give the client/user access to the appropriate resources.
+
+`Did-Auth` flow is similar to the `Sharing VC` flow, but without `VC` exchanges, since `DID` itself should be enough info from the client/user to authenticate.
```

Comment:
could we add a flow diagram here ?

Reply:
it is in place pls [see](https://github.com/affinityproject/affinidi-core-sdk/pull/230/files#diff-77696b96a01de859876a036be6039b21a1fb0faeabd96d660f6aa968ab1694e8R17)